plugins/harvesters/oai/OAIHarvester.inc.php

```diff
@@ -201,6 +201,8 @@ function getSets($harvesterUrl) {
 			return false;
 		}
 
+		if (!$result) return false;
```

Comment:
Can do, although OAIHarvester::getMetadata() and OAIHarvester::getMetadataFormats() also return false on these checks (was following that pattern). I can update/test each of these as well if we'd like to go change the current defaults.